tests/phpunit/tests/Block/Web_Stories_Block.php

```diff
@@ -18,15 +18,30 @@
 namespace Google\Web_Stories\Tests\Block;
 
 use Google\Web_Stories\AMP_Story_Player_Assets;
+use Google\Web_Stories\Assets;
+use Google\Web_Stories\Block\Web_Stories_Block as Block;
+use Google\Web_Stories\Tests\TestCase;
 use WP_Block_Type_Registry;
-use Google\Web_Stories\Tests\Test_Case;
 
 /**
  * @coversDefaultClass \Google\Web_Stories\Block\Web_Stories_Block
  */
-class Web_Stories_Block extends Test_Case {
+class Web_Stories_Block extends TestCase {
+	/** @var Block */
+	private $instance;
+
+	public function setUp() {
+		parent::setUp();
+
+		if ( WP_Block_Type_Registry::get_instance()->is_registered( 'web-stories/embed' ) ) {
+			unregister_block_type( 'web-stories/embed' );
+		}
```

Comment:
Why is this needed in `setUp()` when it is already in `tearDown()`?

Reply:
As you've said yourself, all tests currently simply assume the entire plugin has been loaded and executed

So here I was trying to update the `test_registers_block_type()` test  so that it doesn't simply assume the block is already registered, but instead calls `$embed_block->register()` itself do register it.

I did this because the assertion otherwise failed when tests were executed in random order.

Reply:
IIRC I also tried to make this test use `DependencyInjectedTestCase` when I tried looking into this. So perhaps just some leftover.